### PR TITLE
Hook up some AMQP messages for important actions

### PR DESCRIPTION
**Pull request by spladug**
I reorganized the scraper stuff to take advantage of this newly generalized "update was posted" message. I've tried to make the payloads make sense in and of themselves, but please let me know if I should change them to fit other standards better.

### CODE REVIEW

File: reddit_liveupdate/media_embeds.py [hunk 1]
```diff
@@ -286,24 +278,28 @@ def process_liveupdate_scraper_q():
     def _handle_q(msg):
         d = json.loads(msg.body)
 
+        print d
```
Comment (pwildani) [inline reddit_liveupdate/media_embeds.py]: Debug code

**Review by pwildani — COMMENTED** [review]
Still double checking that the other end of the new events can handle this format, but it should just work.

File: reddit_liveupdate/media_embeds.py (hunk 1 above)
Comment (spladug) [reply reddit_liveupdate/media_embeds.py]: whoops, thanks! fixed.

**Review by pwildani — APPROVED** [review]

**Comment by spladug** [comment]
I've squashed this down and merged it into the `devel` branch which is where I'm staging stuff for deploy. Going to close this out now.

**Comment by spladug** [comment]
:haircut_man: I knew I was missing something with that fullname reformat! I've added a commit which updates the builder to expect proper fullnames in the cached queries and a migration script to rebuild the two static queries live uses.

**Comment by spladug** [comment]
Before:

```
[default@reddit] get LiveUpdateQueryCache['live_events.new.all'];
=> (name=LiveUpdateEvent_zr9qjwucijop, value=[1507757132.404], timestamp=1507757132410382)
=> (name=LiveUpdateEvent_zrixqgmwhqs6, value=[1507868509.721], timestamp=1507868509724331)
=> (name=LiveUpdateEvent_zrmhu6xn4upc, value=[1507911591.291], timestamp=1507911591297326)
=> (name=LiveUpdateEvent_zrmjosh20g5z, value=[1507912213.442], timestamp=1507912213574247)
=> (name=zqqe8rpi0mkq, value=[1507522961.767], timestamp=1507522961771101)
=> (name=zqtva0x9rsb4, value=[1507565011.314], timestamp=1507565011320670)
=> (name=zr9hbq35oulw, value=[1507754029.194], timestamp=1507754029197982)
```

After

```
[default@reddit] get LiveUpdateQueryCache['live_events.new.all'];
=> (name=LiveUpdateEvent_zr9hbq35oulw, value=[1507754029.194], timestamp=1507916681202484)
=> (name=LiveUpdateEvent_zr9qjwucijop, value=[1507757132.404], timestamp=1507916681202484)
=> (name=LiveUpdateEvent_zrixqgmwhqs6, value=[1507868509.721], timestamp=1507916681202484)
=> (name=LiveUpdateEvent_zrmhu6xn4upc, value=[1507911591.291], timestamp=1507916681202484)
=> (name=LiveUpdateEvent_zrmjosh20g5z, value=[1507912213.442], timestamp=1507916681202484)
```

**Comment by spladug** [comment]
:eyeglasses: @bsimpson63 for the recompute script <3

**Review by bsimpson63 — APPROVED** [review]
lgtm

**Review by pwildani — APPROVED** [review]

**Comment by spladug** [comment]
62f9a1c47d33a87eb92696cb0687f266a8f5a5ff...95cce31970bfd1a8cd19044c7177041f55da7fb5

**Pull request closed without merging**